tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -439,6 +464,14 @@ void DebuggerProbesInstrumentationRequester::DetermineReInstrumentProbes(std::se
     }
 }
 
+// Assumes `m_probes_mutex` is held
+bool DebuggerProbesInstrumentationRequester::IsProbeIdExist(const WCHAR* probeId)
```

Comment:
```suggestion
bool DebuggerProbesInstrumentationRequester::ProbeIdExists(const WCHAR* probeId)
```

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.